health-sdk/sdk/src/main/java/net/gini/android/health/sdk/util/extensions/Context.kt

```diff
@@ -24,7 +24,7 @@ internal fun Context.generateBitmapDrawableIcon(icon: ByteArray, iconSize: Int):
                 iconSizePx,
                 true
             )
-            bitmap.recycle()
+            if (bitmap.width != iconSizePx) bitmap.recycle()
```

Comment:
[Suggestion] Could you add please a short comment explaining this fix? It's not obvious why it's needed and would help us understand in the future why it was added. 🙏

Reply:
Added a short explanation, hopefully it makes sense.

Reply:
Thank you, it makes sense 👍 😄